tools/generator/src/Generator+CreateXCSchemes.swift

```diff
@@ -1,11 +1,90 @@
+import PathKit
 import XcodeProj
 
 extension Generator {
     /// Creates an array of `XCScheme` entries for the specified targets.
     static func createXCSchemes(
-        disambiguatedTargets _: [TargetID: DisambiguatedTarget]
-    ) -> [XCScheme] {
-        // GH101: Implement logic to create schemes from targets.
-        return []
+        referencedContainer: String,
```

Comment:
Can you pass in `FilePathResolver` here instead? I don't feel it should be the responsibility of the caller to this function to know how to create a `referencedContainer`.

Reply:
Done.